manager/catalog_manager.go

```diff
@@ -88,24 +88,27 @@ func SetEnv() {
 		for _, value := range catalogURL {
 			value = strings.TrimSpace(value)
```

Comment:
This trim space is not really needed, but after the split on `,` you do need a trimspace there.